tools/python/update_version.py

```diff
@@ -109,7 +108,7 @@ def run(args, cwd):
 
     # upgrade version for onnxruntime-react-native
     run(["npm", "version", version], cwd=os.path.join(js_root, "react_native"))
-    run(["yarn", "upgrade", "onnxruntime-common"], cwd=os.path.join(js_root, "react_native"))
+    run(["npm", "update", "onnxruntime-common"], cwd=os.path.join(js_root, "react_native"))
```

Comment:
this does not work. use `["npm", "install", "--package-lock-only", "--ignore-scripts"]` (refer to a few lines above for onnxruntime-web)